x-pack/plugins/security_solution/public/resolver/store/selectors.ts

```diff
@@ -211,14 +217,9 @@ export const statsTotalForNode = composeSelectors(
 export const visibleNodesAndEdgeLines = createSelector(
   nodesAndEdgelines,
   boundingBox,
-  function (
-    /* eslint-disable @typescript-eslint/no-shadow */
-    nodesAndEdgelines,
-    boundingBox
-    /* eslint-enable @typescript-eslint/no-shadow */
-  ) {
+  function (nodesAndEdgelinesFn, boundingBoxFn) {
     // `boundingBox` and `nodesAndEdgelines` are each memoized.
-    return (time: number) => nodesAndEdgelines(boundingBox(time));
+    return (time: number) => nodesAndEdgelinesFn(boundingBoxFn(time));
```

Comment:
TypeScript did not like the variable name shadowing here, fixing that fixed a type issue below.